src/Compilers/CSharp/Portable/Binder/ContextualAttributeBinder.cs

```diff
@@ -64,5 +66,10 @@ internal Symbol AttributeTarget
                 return _attributeTarget;
             }
         }
+
+        protected override CSDiagnosticInfo SingleTypeNameNotFound(string whereText, DiagnosticBag diagnostics, Location location, AssemblySymbol forwardedToAssembly)
```

Comment:
@leppie In case you haven't seen the comment below, I would prefer if the implementation would be encapsulated into ```Binder.LookupAttributeType``` method without relying on new virtual methods in binder.